docs/hub/organizations-security.md

```diff
@@ -31,3 +31,14 @@ As an organization `admin`, go to the **Members** section of the org settings to
 <img class="hidden dark:block" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/org-members-page-dark.png"/>
 </div>
 
+
+## Viewing members' email address
+
+<Tip warning={true}>
+This feature is part of the <a href="https://huggingface.co/enterprise" target="_blank">Enterprise Hub</a>.
+</Tip>
```

Comment:
need a screenshot that's the most improtant part imo

Reply:
wdym?

Reply:
can we have something like this, but where we can see some (obfuscated or dummy) email addresses (and the other columns too)? 

![image](https://github.com/huggingface/hub-docs/assets/326577/2868ce17-d267-4379-a4eb-e8958fc8bdf7)